try:
    pass
except Exception as e:
    pass
finally:
    pass
